webapp/client/src/index.js

```diff
@@ -30,6 +31,7 @@ const allowedComponents = {
   RevocationPage,
   RevocationSuccessPage,
   DeclarationIncomesPage,
+  DeclarationEDatenPage,
```

Comment:
Maybe the name of the page is a bit confusing because of mixing up german and english.

Reply:
I kept to the old naming what name would you find better? DeclarationEDataPage? Since we have already decided to mix German and English, maybe we should reconsider?

Reply:
Yes, we can stay with mixing it up :)